### DISCUSSION TITLE
fast api support for calling libraries that use MPI?

### DISCUSSION THREAD

**Discussion by NiKeYiGuN** [fastapi/fastapi#11545] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
CMD ["mpirun -np 4 python", "main.py"]
```

### Description

I have a project that uses Fast api and another that requires parallel computation using MPI, is it feasible to start the program via CMD ["mpirun -np 4 python", "main.py"]?

### Operating System

Linux

### FastAPI Version

0.104.1

### Pydantic Version

none

### Python Version

3.11

**Comment by YuriiMotov** [comment]
Let's close this for now to keep the discussions clean and organized.
If you’ve found a solution, please share it here.
If you still need help, feel free to let us know.
